.travis.yml

```diff
@@ -1,19 +1,15 @@
 language: ruby
 rvm:
+  - 2.3.1
   - 2.2
   - 2.1
-  - 2.0
-matrix:
-  include:
-    - # Ruby 1.9
-      rvm: 1.9
-      env: JEKYLL_VERSION=2.0
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
   matrix:
+    - JEKYLL_VERSION=3.3
     - JEKYLL_VERSION=3.0
-    - JEKYLL_VERSION=2.5
+    - JEKYLL_VERSION=2.5.3
```

Comment:
Same as above, but if we want to be extra explicit, I don't think there's any harm :+1: